### PR TITLE
Improve example app styling and hanko-auth defaults

### PR DESCRIPTION
**Pull request by FlxMgdnz**
A stack of small CSS / layout changes to hanko-auth and example app:
- Removed default hanko-auth container styling (most importantly min-width and max-width, border), the element is now fully responsive and has no border etc. Imho it's the better/more flexible solution to leave styling and responsive behaviour of a container around hanko-auth to the website
- Added auth container to example app that contains hanko-auth
- Small fix (overflow: auto) for hanko-auth footer layout
- Changed example app CSS to better align with designs
- Added favicons to example app
- Reduced example app background image size (1MB -> 20KB)

### CODE REVIEW

**Comment by bjoern-m** [comment]
Looks odd to me. This shouldn't be the default?
css1

**Comment by bjoern-m** [comment]
The footer seems to be broken. It seems to stick out of the container if you add a little more height.

css2

**Comment by bjoern-m** [comment]
The files ´hanko-js/README.md` and `hanko-js/src/test.html` do not reflect the changes regarding the CSS variables.

File: hanko-js/src/ui/components/Footer.sass [hunk 1]
```diff
@@ -3,6 +3,7 @@
 .footer
   padding: default.$item-margin
   box-sizing: border-box
+  overflow: auto
```
Comment (bjoern-m) [inline hanko-js/src/ui/components/Footer.sass:6]:
```suggestion
```

**Review by bjoern-m — COMMENTED** [review]
css3

Still isn't quite right, but I figured out what the problem is. I'll push the fix into this branch if you are ok with it?

File: hanko-js/src/ui/components/Footer.sass (hunk 1 above)
[the remark this answers is not in the document]
Comment (FlxMgdnz) [reply hanko-js/src/ui/components/Footer.sass:6]: When trying to bring the CSS closer to what was proposed by Mialena's designs, I encountered a problem with the height of the footer, resulting in the footer links being displayed too close to the bottom of the container. I intended to solve the issue with the overflow "hack" on the footer div: https://stackoverflow.com/a/16568392/15716144

Now after closer inspection, the problem only appeared in combination with my (reverted) removal of `display: flex` from the container. 

Screenshot 2022-08-03 at 19 31 28
Screenshot 2022-08-03 at 19 31 39

With `display: flex` now back again, the overflow hack seems to be no longer required 👍🏼

**Review by bjoern-m — APPROVED** [review]
There we go! Approved!

**Pull request merged**